tag_hierarchy/visitors/filteredhierarchyvisitor.h

```diff
@@ -61,8 +62,36 @@ class FilteredHierarchyVisitor : public boost::default_dfs_visitor
         path_.pop_front();
     }
 
+    void finish_edge(EdgeT e, const TagHierarchyGraph &g) {
+        auto targetVertex = target(e, g);
```

Comment:
target here looks like a free function but it is not qualified with a namespace. If this is indeed a member function it's ok, otherwise it would be good to write out the namespace it is picked from explicitly.

Reply:
Done.